kong/timing/init.lua

```diff
@@ -188,7 +189,7 @@ function _M.log()
     for no, part in ipairs(parts) do
       local msg = string_format("%s id: %s parts: %d/%d output: %s",
                                 constants.REQUEST_DEBUG_LOG_PREFIX,
-                                debug_id, no, nparts, part)
+                                request_id, no, nparts, part)
```

Comment:
This wouldn't be necessary, with @samugi 's feature `ngx.log` already outputs `request_id` unconditionally.

Reply:
done